src/stories/HeatmapVis.stories.tsx

```diff
@@ -75,6 +76,28 @@ NoGrid.args = {
   showGrid: false,
 };
 
+export const LiveData: Story<HeatmapVisProps> = (args): ReactElement => {
```

Comment:
It would be nice to mention the loader in the story name. `LiveDataWithoutLoader` ?

Reply:
nice one 👍